data/data.go

```diff
@@ -2,7 +2,7 @@ package data
 
 import (
 	"crypto/tls"
-	"fmt"
+	// "fmt"
```

Comment:
removed in https://github.com/deis/workflow-manager/pull/49/commits/7e50c5d6a7c6f18492dff2c019417cd3955fb63c